spec/controllers/application_controller_spec.rb

```diff
@@ -13,47 +13,36 @@
     it_behaves_like :most_focus, :category
     it_behaves_like :most_focus, :mood
     it_behaves_like :most_focus, :strategy
-  end
+  end  
 
-  describe 'tag_usage' do
-    it 'is looking for categories tagged nowhere' do
-      new_category = create(:category, user_id: user1.id)
-      result = controller.tag_usage(new_category.id, 'category', user1.id)
-      expect(result[0].length + result[1].length).to eq(0)
+  describe 'moments_stats' do
```

Comment:
You can actually get rid of the tests related to moments_stats since it was moved to moments_helper_spec.rb.